### DISCUSSION TITLE
About the bug of tool file-storage

### DISCUSSION THREAD

**Discussion by Nuyoah94** [he3-app/he3-feedback#353] [Bug Report]
### Version
file-storage 0.1.2

### Environment
Windows v2.0.17

### What is actually happening?
我上传了文件到文件共享内，没有生成链接和QR code

**Comment by lyzhang1999** [comment]
请问你使用的是这个工具吗：https://t.he3app.com/ed3x

**Reply by Nuyoah94** [reply]
是，上传zip文件，不会生成链接和QR Code
